src/QMCDrivers/DMC/DMCUpdatePbyPFast.cpp

```diff
@@ -192,22 +189,27 @@ void DMCUpdatePbyPWithRejectionFast::advanceWalker(Walker_t& thisWalker, bool re
     if(ibar)
     {
       int iat=nonLocalOps.id(ibar);
-      if(!W.makeMoveAndCheck(iat,nonLocalOps.delta(ibar))) return;
-      myTimers[2]->start();
-      Psi.acceptTMove(W,iat,w_buffer);
-#if 0
-      RealType ratio=Psi.ratio(W,iat,dG,dL);
-      W.G += dG;
-      W.L += dL;
-      //thisWalker.R[iat]=W.R[iat];
-      //w_buffer.rewind();
-      //W.copyToBuffer(w_buffer);
-      RealType logpsi = Psi.evaluateLog(W,w_buffer);
+#ifdef ENABLE_SOA
```

Comment:
Good. So Tmoves will need investigating by a future reptation developer/user, and they can't be run accidentally.